src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-currency-step/fixed-deposit-product-currency-step.component.html

```
@@ -32,14 +32,25 @@
-      <mat-label>{{ 'labels.inputs.Currency in multiples of' | translate }}</mat-label>
-      <input type="number" matInput formControlName="inMultiplesOf" />
-      <mat-error>
-        {{ 'labels.inputs.Currency in multiples of' | translate }} {{ 'labels.commons.is' | translate }}
-        <strong>{{ 'labels.commons.required' | translate }}</strong>
-      </mat-error>
-    </mat-form-field>
+    <mat-checkbox
+      labelPosition="before"
+      formControlName="setMultiples"
+      class="flex-98 margin-t"
+      style="align-items: center"
```

HTMLHint (1.8.0)

[warning] Inline style [ 
      style="align-items: center" ] cannot be used.

(inline-style-disabled)